Waveform audio input on `ledmatrix` (#34)
FrameworkComputer/inputmodule-rs: ledmatrix/README.md

inputmodule-control led-matrix --eq 1 2 3 4 5 4 3 2 1
```

###### Input equalizer

> This command generates an equalizer-like visualization of the current audio input. It has only been tested on Linux-based systems running `alsa` or equivalent.

Reviewer: >tested on Linux-based systems running `alsa` or equivalent.

You tested it on all ALSA equivalent systems?  :P

Maybe we should say it supports any system supported by [cpal](https://github.com/RustAudio/cpal).

I can later try it on Windows as well.

Reviewer: ```suggestion
This command generates an equalizer-like visualization of the current audio input.
It supports most platforms - for details, see [documentation of the cpal crate](https://github.com/RustAudio/cpal).
```

You must compile the `inputmodule-control` binary with the `audio-visualization` feature on:

> `cargo clean && cargo build --features audio-visualizations --target x86_64-unknown-linux-gnu -p inputmodule-control`

Reviewer: Any reason to `cargo clean`?

Reviewer: ```suggestion
`cargo build --features audio-visualizations --target x86_64-unknown-linux-gnu -p inputmodule-control`
```

inputmodule-control led-matrix --eq 1 2 3 4 5 4 3 2 1
```

###### Input equalizer

> This command generates an equalizer-like visualization of the current audio input.

Reviewer: ```suggestion
This command generates an equalizer-like visualization of the current audio input (microphone).
```
